### PR TITLE
Issue regarding library path and tests

### PR DESCRIPTION
**Pull request by gmuraru**
## Description
Rename ```federated``` to ```fl``` in the MNIST tutorial and solve indentation issue.
Add ```.gitignore``` file such that datasets from ```examples``` would not be taken into consideration by git.

## Testing
[x] Run the tutorials and validate that they work.

### CODE REVIEW

File: examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb [hunk 1]
```diff
@@ -14,9 +14,40 @@
...
+      "  File \"/home/george/.conda/envs/pysyft-contrib/lib/python3.7/importlib/__init__.py\", line 169, in reload\n",
+      "    _bootstrap._exec(spec, module)\n",
+      "  File \"<frozen importlib._bootstrap>\", line 630, in _exec\n",
+      "  File \"<frozen importlib._bootstrap_external>\", line 728, in exec_module\n",
+      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
+      "  File \"/home/george/contrib/PySyft/examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py\", line 12, in <module>\n",
+      "    from syft.frameworks.torch.federated import utils\n",
+      "ModuleNotFoundError: No module named 'syft.frameworks.torch.federated'\n",
+      "]\n"
+     ]
```
Comment (LaRiffle) [inline examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb]: Is this output expected?

File: examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb [hunk 2]
```diff
@@ -105,30 +136,36 @@
...
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING:root:Torch was already hooked... skipping hooking process\n"
+     ]
+    }
+   ],
```
Comment (LaRiffle) [inline examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb]: same

File: examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py [hunk 3]
```diff
@@ -147,16 +153,16 @@ def evaluate_model_on_worker(
...
+        percentage_0_3 = int(100 * sum(hist_pred[0:4]) / len_dataset)
+        percentage_4_6 = int(100 * sum(hist_pred[4:7]) / len_dataset)
+        percentage_7_9 = int(100 * sum(hist_pred[7:10]) / len_dataset)
+        logger.info(
+            "%s: Percentage numbers 0-3: %s%%, 4-6: %s%%, 7-9: %s%%",
+            model_identifier,
+            percentage_0_3,
+            percentage_4_6,
+            percentage_7_9,
+        )
```
Comment (LaRiffle) [inline examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py]: I think the indentation was correct

File: examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py [hunk 4]
```diff
@@ -258,12 +264,6 @@ def evaluate_model_on_worker(
 
 
 if __name__ == "__main__":
-    # Logging setup
-    FORMAT = "%(asctime)s | %(message)s"
-    logging.basicConfig(format=FORMAT)
-    logger = logging.getLogger("run_websocket_client")
-    logger.setLevel(level=logging.DEBUG)
-
```
Comment (LaRiffle) [inline examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py]: Is this not needed?

File: examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb (hunk 1 above)
Comment (gmuraru) [reply examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb]: Nope. will remove them

File: examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb (hunk 2 above)
Comment (gmuraru) [reply examples/tutorials/advanced/websockets-example-MNIST-parallel/Asynchronous-federated-learning-on-MNIST.ipynb]: Also, will remove this one

File: examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py (hunk 3 above)
Comment (gmuraru) [reply examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py]: Yes, it was ok :(...my bad

File: examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py (hunk 4 above)
Comment (gmuraru) [reply examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py]: I moved at the top of the file - but rerun the examples and it seems it is ok how it is now.

**Pull request merged**